### PR TITLE
Support prefixes for template names

### PR DESCRIPTION
**Pull request by pdufour**
For example if you want to add "/app/" to the start of each template name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ gulp.task('scripts', function() {
       module: 'templates', // optional module name
       extension: 'ngt' // optionally specify what file types to look for
       baseDir: "src/js" // optionally specify base directory for filename
+      prefix: '' // specify a prefix to be added to the filename
```
Comment (javoire) [inline README.md]: maybe `//optionally specify a prefix to be added to the filename`

**Comment by javoire** [comment]
great! could you add a unit test for this as well? there are plenty of them in `test/` for reference

**Comment by pdufour** [comment]
sure thing, thanks.

**Comment by pdufour** [comment]
Done.

**Pull request merged**